kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/BaseSiteService.java

```diff
@@ -538,6 +538,7 @@ public void init()
 			functionManager().registerFunction(SECURE_ADD_SITE);
 			functionManager().registerFunction(SECURE_ADD_USER_SITE);
 			functionManager().registerFunction(SECURE_REMOVE_SITE);
+			functionManager().registerFunction(SOFT_DELETE_SITE);
```

Comment:
Is SOFT_DELETE_SITE a permission, or an event? registerFunction registers permissions that can be associated with Sakai roles.